src/main/java/org/apache/commons/io/IOUtils.java

```diff
@@ -3724,7 +3724,9 @@ public static Writer writer(final Appendable appendable) {
 
     /**
      * Instances should NOT be constructed in standard programming.
+     * @deprecated Will be private in 3.0.
      */
+    @Deprecated
```

Comment:
This is doing more than what's described in the PR title. Maybe worth moving to a separate PR to make it easier to track in the changelog.

Reply:
Thx, I will rollback it

Reply:
Can you remove this deprecation as @kinow requested?

Reply:
Wait... this deprecation is fine. It's slightly out of scope for the PR but it's fine.